src/client/test-run/index.js.mustache

```diff
@@ -101,11 +101,13 @@
         });
 
         runner.on(runner.ACTION_RUN_EVENT, function () {
-            progressPanel.close(true);
+            if(progressPanel)
+                progressPanel.close(true);
         });
 
         runner.on(runner.TEST_FAILED_EVENT, function () {
-            progressPanel.close(false);
+            if(progressPanel)
```

Comment:
Please, add the space after the `if` here and above, it's our code rule
`if (!progressPanel)`